src/services/api.ts

```diff
@@ -6,12 +6,16 @@ import {
   Profile,
   Token,
   OneSignal,
-  SubscriptionInfo
+  SubscriptionInfo,
 } from '../types'
 import { BACKEND_URL } from './constants'
 
-const getPatients = async (token: string): Promise<Patient[]> => {
-  const url = BACKEND_URL + '/patient/'
+const getPatients = async (token: string, fields?: string): Promise<Patient[]> => {
+  let url = BACKEND_URL + '/patient/'
+  if (fields) {
+    url += `?fields=${fields}`
+  }
```

Comment:
I would prefer to have the fields parameter more strongly typed to a `string[]`
And then you can `.join(",")` them